presentation/src/main/java/com/globant/equattrocchio/cleanarchitecture/mvp/view/base/MainActivity.java

```diff
@@ -8,21 +9,28 @@
 import com.globant.equattrocchio.cleanarchitecture.mvp.presenter.ImagesPresenter;
 import com.globant.equattrocchio.cleanarchitecture.mvp.view.ImagesView;
 import com.globant.equattrocchio.data.ImagesServicesImpl;
+import com.globant.equattrocchio.domain.GetJsonUseCase;
 import com.globant.equattrocchio.domain.GetLatestImagesUseCase;
 
 import butterknife.BindView;
 
-public class MainActivity extends AppCompatActivity {
+public class MainActivity extends Activity {
 
     private ImagesPresenter presenter;
-    private GetLatestImagesUseCase getLatestImagesUseCase;
+//    private GetLatestImagesUseCase getLatestImagesUseCase;
+    private GetJsonUseCase getJsonUseCase;
```

Comment:
You only should define as local variable.

Reply:
ok